### DISCUSSION TITLE
Почему в архиве вирусы?

### DISCUSSION THREAD

**Discussion by Amnam01** [bol-van/zapret#831] [Q&A]
![image](https://github.com/user-attachments/assets/b5b3c821-d494-4edc-b8f5-1372d871c923)

**Comment by bol-van** [comment]
Этот вопрос поднимался много раз. Поищите аналогичные топики.
А если коротко : засуньте эти ваши антивирусы с их искусственным и не очень интеллектом куда-нибудь.
Если не хотите, просто удалите софт и забудьте про него. А заодно и всякие кейгены, активаторы, хакерские инструменты и прочие "нехорошие" вещи

**Comment by IndeecFOX** [comment]
Перевожу, например, ответ касперского из скриншота: НЕ ВИРУС: Риск тул (В плохих руках потенциально может нанести вред).
Т.к. утилита берёт ваши данные отправляемые в сеть, видоизменяет их чтобы ТСПУ не обнаружили их, то естественно это внедрение в трафик и естественно об этом предупреждают. Ну как кухонный нож. Потенциально могут и в бочину зарядить))
Так же антивирусникам свойственно ругаться на различные программы активаторы, кейгены, кряки, и много чего ещё что вносит изменение в систему для обхода чего-либо, либо просто нарушает авторские права.
